README.md

```diff
@@ -1,40 +1,148 @@
+<!-- nextcloud-talk-jitzsi-bot README.md -->
+<!-- version: 0.1.1 -->
```

Comment:
Is this header required?

Reply:
> Is this header required?

No, it's not. But it is my habit to tag things, once they are published. It doesn't hurt. Git takes care to be precise.

You are free do decide where to go!